### PR TITLE
update files per PRs: 5026, 5037, 5043, 5049, 5055, 5062

### PR DESCRIPTION
**Pull request by peiying2**
Note, for technology file, one string is being obsolete yet the same string is tagged for a different part. Not sure if I did it correctly.  And if so, do we leverage the translation to the same file? 

* 5026: Dev Extension
* 5037: IH subnav update
* 5041: Enable stub
* 5043: webVR technology

### CODE REVIEW

File: en-US/mozorg/internet-health/shared.lang [hunk 1]
```diff
@@ -17,12 +16,6 @@ Privacy &amp; Security
 Open Innovation
 
 
-## TAG: health-june-campaign-updates
-# Navigation menu (old version)
-;Openness
-Openness
```
Comment (flodolo) [inline en-US/mozorg/internet-health/shared.lang]: Wrong string removed

File: en-US/mozorg/technology.lang [hunk 2]
```diff
@@ -12,19 +13,27 @@ We’re opening up a whole new Internet.
...
 ;Using A-Frame, developers, designers and artists are able to easily create accessible VR experiences.
 Using A-Frame, developers, designers and artists are able to easily create accessible VR experiences.
 
 
-# Obsolete string, do not remove
-;Try it out
-Try it out
-
-
+# String is obsolete. Do NOT remove.
```
Comment (flodolo) [inline en-US/mozorg/technology.lang]: Remove the string here, and leave only the one with the tag.

**Comment by flodolo** [comment]
So:
* Changes to firefox/new are on hold.
* I'll add content for pr 5051.
* Pending clarification on the Thank you page.

We'll need to regroup on Friday when I'm back and see if we're ready to push out strings.

**Comment by flodolo** [comment]
And looks like PR5051 is not relevant anymore.

**Comment by flodolo** [comment]
PR5046 has been closed

**Comment by flodolo** [comment]
I've added strings for PR 5055 and 5062. The latter is pending answers to an issue.

**Comment by flodolo** [comment]
P.S. I really want to get this out of the way as soon as the 5062 is answered. At this point I'm not sure I care about the thank-you one.

**Comment by peiying2** [comment]
Waiting for Eric P to comment on the correction copy.

**Comment by flodolo** [comment]
@peiying2 
Can you do a final check for this? In case, we can send out the strings and unblock it

**Comment by flodolo** [comment]
> Waiting for Eric P to comment on the correction copy.

Which one? independent?

**Comment by peiying2** [comment]
Yeah, Independent needs someone to sign off, but everyone is ready except for someone to step forward to give it ok. I pinged ER but he has not come back with any info.  

For the onboarding one, I have no problem moving forward.

**Pull request merged**